python/trezorlib/tests/device_tests/test_msg_binance_get_address.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from trezorlib import debuglink
+from trezorlib.binance import get_address
+from trezorlib.tools import parse_path
+
+from .common import TrezorTest
+
+
+@pytest.mark.binance
+@pytest.mark.skip_t1  # T1 support is not planned
+class TestMsgBinanceGetAddress(TrezorTest):
+    def test_binance_get_address(self):
+        # data from https://github.com/binance-chain/javascript-sdk/blob/master/__tests__/crypto.test.js#L50
+        debuglink.load_device_by_mnemonic(
```

Comment:
instead: `self._setup_mnemonic(...)`

or better, use new-style tests:
```python
import pytest
from .conftest import setup_client

...

@pytest.mark.binance
@pytest.mark.skip_t1
@setup_client(mnemonic="...")
def test_binance_get_address(client):
	...
```

Reply:
do I understand correctly that with new style tests we drop the class TestMsg... wrapping of tests? anyway I changed it that way and it seems to work ok so just checking.

Reply:
that's correct. classes were necessary for `unittest`-based suite, but we've long since moved to pytest